concourse/model/traits/release.py

```diff
@@ -266,6 +266,15 @@ def __post_init__(self):
         immediately after the release-commit.
         ''',
     ),
+    AttributeSpec.optional(
+        name='post_release_callback',
+        default=None,
+        doc='''
+        an optional callback that is called after release creation. The callback is passed
+        the absolute path to the main repository's work tree via environment variable `REPO_DIR`.
+        No changes in the worktree are possible.
```

Comment:
nit: it would be more accurate to state that changes will be discarded (+ maybe mention `release_callback`. where such changes are honoured)

Reply:
done.